dockerfiles/Dockerfile_r-ver_4.1.0

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:focal
```

Comment:
why prefer `focal` ?  these are technically identical, it seems that 20.04 is more systematically meaningful to users not familiar with ubuntu release chain?

Reply:
This is due to the fact that the process of generating RSPM URLs requires codenames like `focal`, not version numbers like `20.04`.
There are no situations where `20.04` is needed except for specifying the base image, and since `focal` can be used here too, I used `focal` to simplify the process.
It is possible to include `20.04` as before, but it will increase the amount of content to be written in the script.

I didn't think it would cause that much confusion, since Docker Official images often specify the Debian version with a name like `buster`.